### PR TITLE
Updates dbt Cloud doc verbiage and Screenshots

### PR DESCRIPTION
**Pull request by alexbethea22**
## Description & motivation

Updates all outdated dbt Cloud docs and screenshots! 
Most new verbiage added on the Quickstart page to account for project setup flow. Outside of that, a few changes to verbiage here and there, but mostly updates outdated screenshots.

### CODE REVIEW

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md [hunk 1]
```diff
@@ -13,20 +13,20 @@ Connecting your GitHub account to dbt Cloud unlocks exciting and compelling func
...
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/github-app-2.png" title="Installing the dbt Cloud application into an organization."/>
 
 ## Usage Notes
 
 Note that only the Github admin who sets up new dbt Cloud projects needs to connect their GitHub account. It’s not necessary for every dbt Cloud user to connect GitHub (but it can be convenient to do so!)
 
 ## Connect your personal GitHub account
 
-To connect your personal GitHub account, navigate to the [Profile page](https://cloud.getdbt.com/#/profile/). at the bottom of the profile page, you’ll see an “Integrations” section. Under “GitHub”, if your account is already connected, you’ll see "Your user account is linked to a GitHub account with username `<your-github-username>`."
+To connect your personal GitHub account, navigate to your [User Profile](https://cloud.getdbt.com/#/profile/) select the [Integrations section](https://cloud.getdbt.com/#/profile/integrations/) from the left sidebar. Under “GitHub”, if your account is already connected, you’ll see "Your user account is linked to a GitHub account with username <your-github-username>."
```
Comment (clrcrl) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md:30]: I think this is breaking things! You can get around it by using this symbol:

```suggestion
To connect your personal GitHub account, navigate to your [User Profile](https://cloud.getdbt.com/#/profile/) select the [Integrations section](https://cloud.getdbt.com/#/profile/integrations/) from the left sidebar. Under “GitHub”, if your account is already connected, you’ll see "Your user account is linked to a GitHub account with username &lt;your-github-username&gt;."
```

Comment (alexbethea22) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md]: Thank you!!! 🙌

File: website/docs/docs/dbt-cloud/cloud-quickstart.md [hunk 2]
```diff
@@ -10,40 +10,46 @@ The functionality documented here is available in dbt Cloud. Don't have an accou
...
 
-dbt Cloud uses Connections to _connect_ to your database when running dbt jobs. Depending on the type of data warehouse you're using, you'll need to supply different configuration parameters.
+## Connect a repository
+
+dbt Cloud plugs directly into your version control system (GitHub, GitLab, BitBucket, etc) to pull the latest version of your dbt project.
+
+If you've [installed the dbt Cloud application in your GitHub account](cloud-installing-the-github-application), then you will be able to select a repo from your GitHub org using this interface. If you're not using GitHub, or if you haven't installed the integration yet, you can optionally connect a git repository by [providing a git URL](cloud-import-a-project-by-git-url) from the "Git URL" tab.
 
-To create a new Connection, click the Connections link in the left sidebar. From here, click the "New Connection" button to begin creating a new Connection. Next, pick your desired data warehouse type from the `type` dropdown.
+You’ll find the “Managed” tab useful if you do not already have a git repository for your dbt project, you can let [dbt Cloud manage a repository](cloud-using-a-managed-repository) for you.
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/cloud-quickstart.md:32]:
```suggestion
You’ll find the “Managed” tab useful if you do not already have a git repository for your dbt project. With this option, you can let [dbt Cloud manage a repository](cloud-using-a-managed-repository) for you.
```

File: website/docs/docs/dbt-cloud/cloud-quickstart.md [hunk 3]
```diff
@@ -10,40 +10,46 @@ The functionality documented here is available in dbt Cloud. Don't have an accou
...
+Now that dbt Cloud is able to clone your dbt repo and connect to your warehouse, you're ready to create a new job! To create a job, navigate to the Jobs page from the left sidebar, then click the "New Job" button. In the job creation interface, you can specify the environment that the job should use, as well as commands and configuration for your new job.
 
-<Lightbox src="/img/docs/dbt-cloud/3af80f1-Screen_Shot_2019-02-01_at_9.51.54_AM.png" title="An example job definition"/>
+<Lightbox src="/img/docs/dbt-cloud/dbt-quickstart-new-job.png" title="An example job definition"/>
 
 Job schedules can be configured from the job creation interface. You can either schedule your job using the visual editor, or you can enter a custom cron syntax for your job.
 
-<Lightbox src="/img/docs/dbt-cloud/b19ca51-Screen_Shot_2019-02-01_at_12.42.18_PM.png" title="Setting a job schedule"/>
+<Lightbox src="/img/docs/dbt-quickstart-new-job-schedule.png" title="Setting a job schedule"/>
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/cloud-quickstart.md:55]: I think this link isn't rendering in the docs site!

**Pull request merged**